app/controllers/api/vcs.php

```diff
@@ -377,7 +378,7 @@
                 }
                 $owner = $github->getOwnerName($providerInstallationId);
 
-                $providerTargetUrl = $request->getProtocol() . '://' . $request->getHostname() . "/console/project-$projectId/$resourceCollection/$resourceType-$resourceId";
+                $providerTargetUrl = $request->getProtocol() . '://' . $request->getHostname() . "/console/project-$region-$projectId/$resourceCollection/$resourceType-$resourceId";
```

Comment:
Let's ignore current requests's protocol and hostname. Instead, do what we do in worker - env var for http/https, and console env var for hostname

Reply:
Not related to this PR, updating it in the PR for env vars